File: examples/python/07.scripting/05-logger.py
#!/usr/bin/env python3
import logging

log_format = '[%(asctime)s] [%(levelname)s] - %(message)s'
logging.basicConfig(level=logging.DEBUG, format=log_format)
logger = logging.getLogger(__name__)


logger.info("Executando el logger...")
logger.debug("Apuntamos los fallos!")

File: examples/ruby/07.scripting/00.basic/05-logger.rb
#!/usr/bin/env ruby
require 'logger'

logger = Logger.new(STDOUT)
logger.level = Logger::DEBUG

# logger.formatter = proc do |severity, datetime, progname, msg|
#   "#{datetime} [#{severity.ljust(6)}] #{msg}\n"
# end

logger.info("Executando el logger...")
logger.debug("Apuntamos los fallos!")
